src/Minsk/Minsk.csproj

```diff
@@ -1,12 +1,14 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
+    <IsPackable>false</IsPackable>
+    <IsPublishable>false</IsPublishable>
```

Comment:
Setting these will prevent us doing unnecessary pack/publish for the specified projects in local/CI build when we do it on the entire solution.